_data/changelogs/component-memorable-date.yml

```diff
@@ -2,6 +2,15 @@ title: Memorable date
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Removed numeric representation of months from `select`.
```

Comment:
**suggestion (optional):** Maybe add `component` or `element` to the summary. Ending on select reads odd to me 🤔 

```suggestion
  summary: Removed numeric representation of months from the `select` element.
```